elisp/defs.bzl

```diff
@@ -280,7 +280,6 @@ def _elisp_test_impl(ctx):
     ]
 
 elisp_toolchain = rule(
-    implementation = _elisp_toolchain_impl,
```

Comment:
Please revert these formatting changes.

Reply:
Done. This was applied by vim running buildifier on all bzl files automatically, sorry about that.